src/mono/mono/mini/simd-intrinsics.c

```diff
@@ -1944,32 +1944,10 @@ emit_vector64_vector128_t (MonoCompile *cfg, MonoMethod *cmethod, MonoMethodSign
 		return NULL;
 #endif
 
-// FIXME: This limitation could be removed once everything here are supported by mini JIT on arm64
+// FIXME: Support Vector64 for mini JIT on arm64
 #ifdef TARGET_ARM64
-	if (!COMPILE_LLVM (cfg)) {
+	if (!COMPILE_LLVM (cfg) && (size != 16))
 		return NULL;
-		if (size != 16)
```

Comment:
Unrelated ?

Reply:
We haven't tested Vector64 yet. This is a guard to make sure only Vector128 could go through